### ISSUE TITLE
VxMark: modify "Print My Ballot" UX to avoid accidental or overeager casting

### ISSUE THREAD

**Issue by arsalansufi** [votingworks/vxsuite#5324]
Here's a mock confirm ballot pop up. People have been accidentally printing their ballot and a pop-up screen could stop people from accidentally printing their ballot.

Issue automatically created from Slack feedback reported by ryan.aparicio: 
https://votingworks.slack.com/archives/CJU9MSC6S/p1724427038018389?thread_ts=1724427038.018389&cid=CJU9MSC6S

**Comment by adghayes** [comment]
Suggested fix: only change color of primary action button to a colored (truly primary) button until the user scrolls fully through the screen.
